### PR TITLE
Restore blendball to original functionality

### PR DESCRIPTION
**Pull request by KevinDaGame**
## What type of PR is this? (check all applicable)

- [ ] Feature
- [ ] Refactor
- [x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
Restore the original functionality of the blendball brush. I know this is a hacky fix, but for now it should be fine

### CODE REVIEW

File: VoxelSniperCore/src/main/java/com/github/kevindagame/brush/polymorphic/operation/BlendOperation.kt [hunk 1]
```diff
@@ -2,15 +2,19 @@ package com.github.kevindagame.brush.polymorphic.operation
...
 
 class BlendOperation: PolyOperation(listOf(PolyOperationType.BLOCK)) {
     override fun apply(
         brushSize: Int,
         brush: PolyBrush,
+        snipeAction: SnipeAction,
         excludeAir: Boolean,
         excludeWater: Boolean
     ): Array<Array<Array<VoxelMaterial>>> {
+        val doExcludeAir = snipeAction == SnipeAction.GUNPOWDER
```
Comment (Lennart99) [inline VoxelSniperCore/src/main/java/com/github/kevindagame/brush/polymorphic/operation/BlendOperation.kt:17]: Do we than still need the excludeAir argument?

**Review by Lennart99 — APPROVED** [review]
Should we also remove EXCLUDEAIR from PolyPropertiesEnum?

File: VoxelSniperCore/src/main/java/com/github/kevindagame/brush/polymorphic/operation/BlendOperation.kt (hunk 1 above)
Comment (KevinDaGame) [reply VoxelSniperCore/src/main/java/com/github/kevindagame/brush/polymorphic/operation/BlendOperation.kt:17]: Perhabs not, but for now I'll leave it there

Comment (Lennart99) [reply VoxelSniperCore/src/main/java/com/github/kevindagame/brush/polymorphic/operation/BlendOperation.kt:17]: sure

**Pull request merged**